config.sh

```diff
@@ -0,0 +1,70 @@
+function pre_build {
+    echo "Starting pre-build"
+
+    # Any stuff that you need to do before you start building the wheels
+    # Runs in the root directory of this repository.
+
+    # https://github.com/matthew-brett/multibuild/pull/146
+    yum install -y rsync
+
+    echo 'Building fftw'
+    # Taken from: https://github.com/conda-forge/fftw-feedstock/blob/master/recipe/build.sh
+    export CFLAGS="-O3 -fomit-frame-pointer -fstrict-aliasing -ffast-math"
+
+    # TODO: make this a loop:
+    # single
+    echo 'Building fftw: single'
```

Comment:
@matthew-brett What do you think of factoring out building fftw as one of the [recipes](https://github.com/matthew-brett/multibuild/blob/42fdd0c87d26e62655d6f64b86fba2d1e37fb85b/library_builders.sh) in `multibuild`? There is at least one other python wheel that will need it: https://github.com/jyhmiinlin/pynufft.